torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -78,6 +82,31 @@ def _get_default_op_backend_config(op, dtype_configs):
         torch.nn.functional.layer_norm,
     ]]
 
+
+def _binary_op_observation_type_getter(quantize_handler):
```

Comment:
is this exposed to users?  Is there a way to not expose things like a quantize_handler input?

have we considered just modeling add_scalar and add_tensor separately in these configs?

Reply:
this is a custom function to overwrite the observation type, basically user can write a function to return the observation type based on the properties of root node. currently we use quantize_handler object, but this is just a convenience so that we don't need to spell out all relevant components here like root_node, modules etc., we can also expose root_node, modules, but I think that is not very scalable, since maybe there are other information that is relevant for user to decide the observation type.

Reply:
quantize_handler is just a place holder for the information that is related to the matched node here, we may rename it later

Reply:
it would be simpler to only allow the user to provide static configs, adding an extension point of a python function adds significant complexity. Can we allow different behaviors based on number of tensor arguments, and not expose the logic which calculates the number of tensor arguments?

Reply:
yeah sounds good

Reply:
will continue the discussion in the other comment